client/package.json

```diff
@@ -42,5 +42,10 @@
     "prettier": "^3.1.0",
     "vite": "^5.0.0"
   },
-  "packageManager": "yarn@4.6.0"
+  "packageManager": "yarn@4.6.0",
+  "overrides": {
+    "react-tournament-brackets": {
+      "styled-components": "$styled-components"
```

Comment:
I believe I had to bump `styled-components` for some reason or the `npm` version I'm using is detecting that it conflicts with the version `styled-components` required from `react-tournament-brackets`. This seems to work anyway.

Reply:
Yeah, seems to be a version mismatch in the devdependencies not the dependencies that matter. I might look into resolving this but it's fine. :)